vendor/gopkg.in/fsnotify.v1

```diff
@@ -0,0 +1 @@
+Subproject commit 875cf421b32f8f1b31bd43776297876d01542279
```

Comment:
Not convinced we want this as a subproject; do the thing that you're doing with `tomb` instead?